code/datums/traits/negative.dm

```diff
@@ -120,17 +122,23 @@
 				heirloom_type = /obj/item/storage/toolbox/mechanical/old/heirloom
 			if("Barber")
 				heirloom_type = /obj/item/handmirror
+			if("Psychiatrist")
+				heirloom_type =  pick(subtypesof(/obj/item/toy/plush))
 			if("Stage Magician")
 				heirloom_type = /obj/item/gun/magic/wand
+			if("VIP")
+				heirloom_type = /obj/item/pda/clear				
 			//Security/Command
 			if("Captain")
-				heirloom_type = /obj/item/reagent_containers/food/drinks/flask/gold
+				heirloom_type = pick(/obj/item/reagent_containers/food/drinks/flask/gold, /obj/item/phone)
+			if("Head Of Personnel")
+				heirloom_type = pick(/obj/item/toy/figure/ian, /obj/item/phone)
 			if("Head of Security")
-				heirloom_type = /obj/item/book/manual/wiki/security_space_law
+				heirloom_type = pick(/obj/item/book/manual/wiki/security_space_law, /obj/item/phone)
 			if("Warden")
 				heirloom_type = /obj/item/book/manual/wiki/security_space_law
 			if("Security Officer")
-				heirloom_type = pick(/obj/item/book/manual/wiki/security_space_law, /obj/item/clothing/head/beret/sec)
+				heirloom_type = pick(/obj/item/clothing/head/beret/sec, /obj/item/clothing/head/helmet/old)
```

Comment:
old helmet is worse than useless. just remove the spacelaw book

Reply:
Isn't that the point of heirlooms, to be a burden?